docs/getting_started.md

```diff
@@ -203,6 +203,167 @@ See these resources for more information on clustering:
  - [Quantifying clustering behavior in Orchestrating Single Cell Analysis](https://bioconductor.org/books/release/OSCA.advanced/clustering-redux.html#quantifying-clustering-behavior)
 
 
+### Cell type annotation
```

Comment:
Generally, I think we want to add a little bit more context to this section. I would describe the references used in this first section. I would also mention the supplemental cell type report and that it's a good resource for evaluating the provided cell type annotations.

Reply:
I think what you added is good!